aerospike/tests/test_unit.py

```diff
@@ -3,7 +3,7 @@
 import mock
 import pytest
 
-from datadog_checks import aerospike
+from datadog_checks.base import aerospike
```

Comment:
Let's try to do this instead
```python
from datadog_checks.aerospike import AerospikeCheck
```

Reply:
Hum, just relied on the `ddev validate imports --autofix`.

Seems not work correctly for this case.

Reply:
@FlorianVeaux @mgarabed I fixed the validator instead to skip this case.

Reply:
Yeah, the logic was to just add `base` in there, but didn't account for the imports already supported from just `datadog_checks`.